inc/namespace.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Altis\Chassis;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	add_filter( 'qm/output/file_path_map', __NAMESPACE__ . '\\set_file_path_map', 1 );
+}
+
+/**
+ * Enables Query Monitor to map paths to their
+ * original values on the host.
+ *
+ * @param array $map an associative array of folders as keys, and their mappings as values
+ * @return array the adjusted mapping of folders
+ */
+function set_file_path_map( array $map ) : array {
+	// Chassis and Local Server
+	if ( ! file_exists( '/etc/chassis-constants' ) ) {
+		return $map;
+	}
+	$json_string = file_get_contents( '/etc/chassis-constants' );
+	$data = json_decode( $json_string, true );
+	if ( empty( $data['synced_folders'] ) ) {
```

Comment:
I'd prefer an explicit `empty( $data )` here, as it's a different code path.

Reply:
I'd still need an additional guard to ensure that the `synced_folders` value is present, so I don't see what the additional `empty` call provides

Reply:
There's two ways `json_decode` can fail conceptually:
1. JSON fails to parse -> `$data` is `null`, and `json_last_error()` is not `JSON_ERROR_NONE`
2. JSON parses, but doesn't contain a `synced_folders` key

Using one check works here, but two (where one is technically redundant) is more explicit about the failure cases. Additionally, I'm squeamish about using array indexes on `null`, even if it is within a language construct, so would prefer to avoid that.